bin/impl/redeploy.sh

```diff
@@ -15,7 +15,18 @@
 # limitations under the License.
 
 # stop if any command fails
-set -e  
+set -e
+
+function verify_checksum() {
+  tarball=$1
+  expected_md5=$2
```

Comment:
my thinking is that the sanity check on expected_md5 makes the equality check below a bit more robust